api/src/wfl/module/wgs.clj

```diff
@@ -106,7 +106,7 @@
                    (assoc input-key in-gs)
                    (assoc :destination_cloud_path (str out-gs out-dir))
                    (assoc :references (make-references ref-prefix))
-                   #_(merge fingerprinting) ;; Uncomment to enable fingerprinting
+                   (merge fingerprinting) ;; Uncomment to enable fingerprinting
```

Comment:
remove the comment too!

Reply:
Oops, my bad. Guess this is staying commented anyway for right now? I'm still trying to sort out this ticket, I reverted the commit that uncommented this